components/TextControls/tests/unit/OutlinedTextAreas/MDCOutlinedTextAreaTests.m

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialAvailability.h"
+#import "MaterialTextControls+OutlinedTextAreas.h"
+
+@interface MDCOutlinedTextAreaTests : XCTestCase
+@end
+
+@implementation MDCOutlinedTextAreaTests
+
+#pragma mark Tests
+
+- (void)testOutlineColorDefaults {
+  // Given
+  CGRect textAreaFrame = CGRectMake(0, 0, 130, 40);
+  MDCOutlinedTextArea *textArea = [[MDCOutlinedTextArea alloc] initWithFrame:textAreaFrame];
+  UIColor *defaultOutlineColor = UIColor.blackColor;
+#if MDC_AVAILABLE_SDK_IOS(13_0)
+  if (@available(iOS 13.0, *)) {
+    defaultOutlineColor = UIColor.labelColor;
+  }
+#endif  // MDC_AVAILABLE_SDK_IOS(13_0)
+
+  // Then
+  XCTAssertEqualObjects(defaultOutlineColor,
+                        [textArea outlineColorForState:MDCTextControlStateNormal]);
+  XCTAssertEqualObjects(defaultOutlineColor,
+                        [textArea outlineColorForState:MDCTextControlStateEditing]);
+  XCTAssertEqualObjects([defaultOutlineColor colorWithAlphaComponent:(CGFloat)0.60],
+                        [textArea outlineColorForState:MDCTextControlStateDisabled]);
+}
+
+- (void)testOutlineColorAccessors {
+  // Given
+  CGRect textAreaFrame = CGRectMake(0, 0, 130, 40);
+  MDCOutlinedTextArea *textArea = [[MDCOutlinedTextArea alloc] initWithFrame:textAreaFrame];
+  UIColor *outlineColorNormal = [UIColor blueColor];
```

Comment:
I think we typically use dot syntax can we update these to stick with the norm.

Reply:
Done.